scripts/test/procedures.py

```diff
@@ -44,14 +43,15 @@ def run_pyright_dist():
     cmd = ["pyright", "tests"]
     subprocess.run(cmd, check=True)
 
+
 def uninstall_dist():
     cmd = ["pip", "uninstall", "-y", "pandas-stubs"]
     subprocess.run(cmd, check=True)
 
 
 def restore_src():
-    cmd = ["git", "checkout", "HEAD", "pandas-stubs"]
-    subprocess.run(cmd, check=True)
+    if Path(r"_pandas-stubs").exists():
```

Comment:
Can you abort here if `_pandas-stubs` somehow already exists?

Reply:
Done